sdk/servicebus/Azure.Messaging.ServiceBus/src/Diagnostics/ServiceBusEventSource.cs

```diff
@@ -190,6 +190,8 @@ private ServiceBusEventSource(string eventSourceName) : base(eventSourceName, Ev
         internal const int RequestAuthorizationCompleteEvent = 106;
         internal const int RequestAuthorizationExceptionEvent = 107;
 
+        internal const int ProcessorClientClosedExceptionEvent = 108;
```

Comment:
I wasn't sure on the preference to keep things grouped by functional area or preserve numeric ordering.   If preferred, I can push this up with the other processor-related events.

Reply:
I think this is fine so it is clear what the last number is.